source/features/forked-to.tsx

```diff
@@ -107,7 +115,7 @@ async function init(): Promise<void | false> {
 	}
 
 	// This feature only applies to users that have multiple organizations, because that makes a fork picker modal appear when clicking on "Fork"
-	const hasOrganizations = await elementReady('details-dialog[src*="/fork"] include-fragment');
+	const hasOrganizations = await elementReady('details-dialog[src*="/fork"]');
```

Comment:
Works without `elementReady` too 👍🏾

Reply:
Could you test that it works with a throttled connection too? (You can do it with the developers tools, in Firefox it's under the "Network" tab).

Often I find that pages load quickly enough for a plain `select` to work most of the time ­­— except it will randomly break when loading become slightly slower for any reason.

Reply:
`select` in all places (the `forkedCounter`, `forkedContainer` and `hasOrganizations`) does not work with Regular 2G or lower - using `elementReady` in all three places for now. Let me know if I should use `select` instead.

Reply:
I've dropped one of the two `elementReady`, it seems to work well enough without it.